### PR TITLE
Add sogou ads support for amp

### PR DESCRIPTION
**Pull request by spliu78**
Add sogou ads support for amp.

### CODE REVIEW

**Comment by spliu78** [comment]
I signed it!

**Comment by spliu78** [comment]
@jridgewell 
Is there any command that can revert yarn.lock?
I've tried `git log yarn.lock` and then `git reset xxx yarn.lock` but it doesn't seem like work.
I'm sorry that I'm new about git.

**Comment by jridgewell** [comment]
```js
git reset yarn.lock
git checkout master -- yarn.lock
```

**Comment by spliu78** [comment]
@jridgewell Done.

**Comment by spliu78** [comment]
@jridgewell Is there anything left to do?

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -282,6 +282,7 @@ See [amp-ad rules](https://github.com/ampproject/amphtml/blob/master/extensions/
 - [Smart AdServer](../../ads/smartadserver.md)
 - [smartclip](../../ads/smartclip.md)
 - [Sortable](../../ads/sortable.md)
+- [sogou_ad](../../ads/sogou_ad.md)
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: `[Sogou Ad]`

File: yarn.lock [hunk 2]
```diff
@@ -48,10 +48,6 @@ acorn-jsx@^3.0.0:
   dependencies:
     acorn "^3.0.4"
 
-acorn@4.0.4, acorn@^4.0.0:
```
Comment (lannka) [inline yarn.lock]: can you revert?

File: 3p/integration.js [hunk 3]
```diff
@@ -153,6 +153,7 @@ import {slimcutmedia} from '../ads/slimcutmedia';
 import {smartadserver} from '../ads/smartadserver';
 import {smartclip} from '../ads/smartclip';
 import {sortable} from '../ads/sortable';
+import {sogou_ad} from '../ads/sogou_ad';
```
Comment (lannka) [inline 3p/integration.js]: to follow the naming convention: `sogouad`

**Comment by lannka** [comment]
> If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits.

@spliu78 pls follow the CLA bot and resolve the issue.

**Comment by spliu78** [comment]
I signed it!

**Comment by spliu78** [comment]
@lannka done

**Comment by lannka** [comment]
@spliu78 could you address my previous comments?

**Comment by spliu78** [comment]
@lannka I'm sorry that I'm not using git well.
 which version should I revert to ? Or what command should I try?
I've try a lot times.
Can I just copy this one https://raw.githubusercontent.com/ampproject/amphtml/master/yarn.lock ?But it looks the same as my working copy.

**Comment by lannka** [comment]
@spliu78 take a look at [this post](https://stackoverflow.com/questions/2733873/reverting-a-single-file-to-a-previous-version-in-git) for how to revert a file into a previous version in git. 

After that, you can do a force push
```
git push --force
```

**Comment by spliu78** [comment]
I use this version: 6b1873bdb456b7ee333239bdb601a61ef7ddd0cd
@lannka It's that OK now?

**Comment by lannka** [comment]
@spliu78 yarn.lock seems to be gone. but would you address my comment regarding to the renaming

**Comment by lannka** [comment]
@spliu78 pls fix the travis error

**Comment by spliu78** [comment]
@lannka Done.

**Pull request merged**